src/main.js

```diff
@@ -0,0 +1,82 @@
+//Asynchronusly fetches data from api
+const getData = async () => {
+    try {
+        const response = await fetch('https://randomuser.me/api/?results=20'); // Fetch data for 10 users
+        const data = await response.json();
+        return data.results;
+    } catch (error) {
+        console.error('Error fetching data:', error);
+    }
+}
+
+//Updates DOM elements, gets data as a parameter and displayes it in the browser
+const updateDOM = (data) => {
+    const mainElement = document.querySelector('.main');
+
+    //iterates through fethed data
+    data.forEach(user => {
+        const userCard = document.createElement('div');
+        userCard.className = 'card';
+        mainElement.appendChild(userCard);
+
+        //user image
+        const image = document.createElement('img');
+        const imgUrl = user.picture.large;
+        const imgAlt = `${user.name.first} ${user.name.last} photo`;
+        image.src = imgUrl;
+        image.alt = imgAlt;
+        userCard.appendChild(image);
+
+        //unordered list to display user's data
+        const unorderedList = document.createElement('ul');
```

Comment:
31-ic mincev 67 toxer@ karox eiq grel veli karc orinak ayspes const userCard = document.createElement('div');
document.body.appendChild(userCard);

const userInfo = {
  'Firstname': user.name.first,
  'Lastname': user.name.last,
  'Gender': user.gender,
  'Email': user.email,
  'Phone number': user.cell,
  'Username': user.login.username,
  'Signed in at': user.registered.date
};

const unorderedList = document.createElement('ul');
userCard.appendChild(unorderedList);

for (const key in userInfo) {
  const listItem = document.createElement('li');
  listItem.textContent = `${key}: ${userInfo[key]}`;
  unorderedList.appendChild(listItem);
}

Reply:
Շնորհակալ եմ խորհուրդների համար, կդիտարկեմ տարբերակները անպայման, առաջին հայացքից շատ հետաքրքիրա երևում